portfolio/src/main/webapp/piano.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Piano</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="piano.js"></script>
+  </head>
+<body>
+  <div id="content">
+
+    <p data-key="65" class = "key" data-note="C">C</p>
```

Comment:
The `data-keys` are keycodes, but that is never documented or mentioned in the code. I think the point of having such codes is that you can set up a keyboard listener, and directly know from a given key, its keycode, and then find the corresponding audio file to play.

However the change doesn't add any keyboard controls. Changes should be self-contained, and one should be able to understand a change in isolation. So if you're planning to add keyboard controls in the future, I would document this and write a comment explaining that the data-key fields are set to these values, because we'll implement keyboard controls in the future. 

If you are not planning to add keyboard controls, then the data-key values are very confusing and are adding unnecessary complexity, and they could simply be the string of the key itself (e.g. `data-key="A"` for the 'A' key, `data-key="A#"` for the 'A#' key, etc.).